cpanfile

```diff
@@ -3,6 +3,7 @@
 
 requires 'perl', '5.32.0';
 
+requires 'Array::PrintCols';
```

Comment:
Hmm. Why do we need this dependency? I know they are considered archaic, but Peel has formats which are useable to print table data. It's what i used to print the data about the available templates. They still work, although they lack flexibility with respect to the number of printed columns and their width.

Reply:
We have tons of roles. Printing them vertically sorted is more usual than horizontally.